components/pool/PoolForm.tsx

```diff
@@ -159,6 +159,10 @@ const PoolForm: React.FC<{}> = () => {
 			if (coreAmount <= 0) {
 				setCoreAmount(0);
 			} else {
+				if (formatBalance(coreAmount) === "<0.0001") {
+					setCoreError("Amount too low");
```

Comment:
should use `coreAmount` as number to detect if amount is too low instead of relying on the output of `formatBalance`. This check will fail silently if later one we decide to update the output to `< 0.0001` (with space) for example.